### PR TITLE
updated Law and a few periodicals per Chris's changes

### PR DESCRIPTION
**Pull request by ruthtillman**
it ended up cleaning up spacing a bit when I ran it through prettify

### CODE REVIEW

File: app/javascript/psulib_blacklight/javascripts/libraries_locations.json [hunk 1]
```diff
@@ -1,716 +1,716 @@
...
+    "ONHOLD-SV": "On hold at Penn State Shenango",
+    "ONHOLD-WB": "On hold at Penn State Wilkes-Barre",
+    "ONHOLD-WD": "On hold for World Campus",
+    "ONHOLD-WS": "On hold at Penn State Scranton",
+    "ONHOLD-YK": "On hold at Penn State York",
+    "ONLINE-DSL": "Online Content (Law Schools)",
+    "ONLINE": "Online Content",
+    "ON-ORDER": "Being acquired by the library",
+    "OPAQUE-NML": "2nd Floor Paterno - Microprint",
+    "OPENRES-AC": "Architecture & Landscape Arch, Stuckeman Bldg â€“ Service Desk",
```
Comment (banukutlu) [inline app/javascript/psulib_blacklight/javascripts/libraries_locations.json]: @ruthtillman one more weird character here in this line

Comment (banukutlu) [reply app/javascript/psulib_blacklight/javascripts/libraries_locations.json]:
```suggestion
    "OPENRES-AC": "Architecture & Landscape Arch, Stuckeman Bldg – Service Desk",
```

Comment (banukutlu) [reply app/javascript/psulib_blacklight/javascripts/libraries_locations.json]: @ruthtillman I just updated it with a `–`, hope that was correct

Comment (ruthtillman) [reply app/javascript/psulib_blacklight/javascripts/libraries_locations.json]: yep thank you.

File: app/javascript/psulib_blacklight/javascripts/libraries_locations.json [hunk 2]
```diff
@@ -1,716 +1,716 @@
...
+    "PSUARCHIVE": "Univ. Archives, 1st Floor Paterno",
+    "PSUARC-MSS": "Univ. Archives, 1st Floor Paterno, Univ. Archives & Mss",
+    "PSUPRESS": "Univ. Archives, 1st Floor Paterno, Univ. Press Coll.",
+    "PUGH": "Univ. Archives, 1st Floor Paterno, Pugh Collection",
+    "PURPLEQUIL": "Univ. Archives, 1st Floor Paterno, Purple Quill Coll.",
+    "RAMP-DP": "Penn State Law (UP) - Carrel Ramp",
+    "RAREBKS-HB": "Harrisburg - Rare Books - Special Coll - 3rd Floor",
+    "RARE-BOOK": "Rare Books & Mss, 1st Floor Paterno",
+    "RARE-CHILD": "Rare Books & Mss, 1st Floor Paterno, Children's Book Coll.",
+    "RARE-DN": "Dickinson Law (Carlisle) - Rare Book R
```
Comment (banukutlu) [inline app/javascript/psulib_blacklight/javascripts/libraries_locations.json]:
```suggestion
    "RESERVE-AC": "Architecture & Landscape Arch, Stuckeman Bldg – Open Reserve",
```

@ruthtillman replacing this one with a `–` as well

Comment (banukutlu) [inline app/javascript/psulib_blacklight/javascripts/libraries_locations.json]:
```suggestion
    "RESERVE-AC": "Architecture & Landscape Arch, Stuckeman Bldg – Open Reserve",
```

@ruthtillman replacing this one with a `–` as well

Comment (banukutlu) [inline app/javascript/psulib_blacklight/javascripts/libraries_locations.json]:
```suggestion
    "RESERVE-AC": "Architecture & Landscape Arch, Stuckeman Bldg – Open Reserve",
```
@ruthtillman replacing this one with a `–` as well

**Pull request merged**